docs/administration/back_office/search_sorting.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Add a "sort by" method to the Back Office search result page.
+---
+
+# Customize search sorting
+
+To add an entry to the "Sort by" to the Back Office search result page, create a service implementing the `SortingDefinitionProviderInterface` and tagged `ibexa.search.sorting_definition.provider`.
```

Comment:
Let's use FQCN when we introduce symbol: `Ibexa\Contracts\Search\SortingDefinition\SortingDefintionProviderInterface`

Reply:
```suggestion
To add an entry to the "Sort by" to the Back Office search result page, create a service implementing the `Ibexa\Contracts\Search\SortingDefinition\SortingDefintionProviderInterface` and tagged `ibexa.search.sorting_definition.provider`.
```